src/makeCalculator.js

```diff
@@ -4,7 +4,23 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate: function (sv, x) {
+      this.result = sv(this.result, x);
+
+      return this;
+    },
+    reset: function () {
+      this.result = 0;
+
+      return this;
```

Comment:
You can use method shorthand syntax here as well. Replace `reset: function ()` with `reset()`.